packages/kbn-i18n/GUIDELINE.md

```diff
@@ -385,6 +388,12 @@ Splitting sentences into several keys often inadvertently presumes a grammar, a
 
   If this group of sentences is separated it’s possible that the context of the `'it'` in `'close it'` will be lost.
 
+### Large paragraphs
+
+Try to avoid having to use large paragraphs of text. These are difficult to maintain and often need slight changes when the information they portray becomes out of date. 
```

Comment:
```suggestion
Try to avoid using large paragraphs of text. These are difficult to maintain and often need slight changes when the information they portray becomes out of date. 
```